tracing/src/main/java/com/palantir/remoting3/tracing/CloseableTracer.java

```diff
@@ -38,12 +38,12 @@ private CloseableTracer() { }
      * Opens a new {@link SpanType#LOCAL LOCAL} span for this thread's call trace, labeled with the provided operation.
      */
     public static CloseableTracer startSpan(String operation) {
-        Tracer.startSpan(operation);
+        com.palantir.tracing.Tracer.startSpan(operation);
```

Comment:
can we just be like com.palantir.tracing.CloseableTracer.startSpan?

Reply:
i kind of prefer using `com.palantir.tracing.Tracer.startSpan(operation);` to match up against `com.palantir.tracing.Tracer.fastCompleteSpan();` few lines below since we can't get the instance of the `com.palantir.tracing.CloseableTracer`.